src/pykoala/ancillary.py

```diff
@@ -14,21 +14,54 @@
 # =============================================================================
 # Astropy and associated packages
 # =============================================================================
-from astropy.io import fits
+#from astropy.io import fits
 # =============================================================================
-
 # =============================================================================
 # PyKOALA modules
 # =============================================================================
 from pykoala.cubing import Cube, build_wcs, build_cube
-
-
+# =============================================================================
+# =============================================================================
+# =============================================================================
+# ANGEL modules                                                            #!!!
+# =============================================================================
+# =============================================================================
+# =============================================================================
 def vprint(*arg, **kwargs):
     """
     Prints the arguments only if verbose=True.
     """
-    if kwargs['verbose']:
-        print(*arg)
+    try:
+        if kwargs['verbose']:
+            print(*arg)
+    except Exception:
+        do_nothing = 0                    #!!! Ángel: Una ayudita aquí... si no quiero que haga nada si falla, ¿qué pongo?
+# =============================================================================
+# =============================================================================
+def vplot(*arg, **kwargs):
+    """
+    Check kwarguments and return True if plot=True.
+    """
+    try:
+        if kwargs['plot']:
+            return True
+        else:
+            return False
+    except Exception:
+        return False
```

Comment:
This is an overkill. You already have a boolean variable `plot` telling you whether to plot or not, I don't see the need of including an extra call to this function.

Reply:
I created this function BEFORE you told me the trick of `kwargs.get('variable')`, also testing how kwargs work. Last weekend I implemented another function "check_kwargs" the did precisely that, I replaced all in my code but didn't pay attention to `vplot()`, that indeed I was going to remove. Deleting it.